jsx/Form.js

```diff
@@ -1497,9 +1538,11 @@ class ButtonElement extends Component {
       <div className="row form-group">
         <div className={this.props.columnSize}>
           <button
+            {...(this.props.id ? {id: this.props.id} : {})}
             name={this.props.name}
             type={this.props.type}
             className={this.props.buttonClass}
+            {...(this.props.style ? {style: this.props.style} : {})}
```

Comment:
Similar to the comment above but for adding a style. I'm assuming there's no right way of doing it. I thought this way I came up with is slick.

Reply:
I think it may be the way that's written that is confusing. The following might be a bit more straight forward.
`style={this.props.style ? this.props.style : {}}`
`style={this.props.style && this.props.style}`

 I'm also not sure if you have to do the check in the first place. I believe if you pass a value of `null` to a prop, it does not get passed to the child.

Reply:
@HenriRabalais Okay see latest changes. Previously I was worried if not supplied it would be added but react handles that.